src/effect.js

```diff
@@ -87,7 +87,7 @@ define(function (require) {
 	 *	@param {Float}
 	 */
 	p5.Effect.prototype.drywet = function(fade){
-		this._drywet.fade.value = fade;
+		this._drywet.fade.value = fade || this._drywet.fade.value;
```

Comment:
if `fade` is zero, this technique treats `0` as "falsey". Another approach would be 

If zero should be allowed, then we really want to check for `typeof fade !== 'undefined'`